Makefile

```diff
@@ -9,6 +9,7 @@ LUA_VERSION=5.1
 
 LUA_PATH = deps/share/lua/$(LUA_VERSION)/?.lua;deps/share/lua/$(LUA_VERSION)/?/init.lua
 LUA_PATH += ;lua/?.lua;lua/?/init.lua
+LUA_PATH += ;$(shell lua$(LUA_VERSION) -e 'print(package.path)')
 export LUA_PATH
 
 export LUA_CPATH = deps/lib/lua/$(LUA_VERSION)/?.so
```

Comment:
[Nope](https://github.com/dundargoc/nvim-package-specification/runs/5235640254?check_suite_focus=true) :/

Reply:
Okay, the problem is most likely this: `no file 'lua/busted/init.lua '`
It's because of how make concatenates variables with `+=`, it adds a space.
```make
export LUA_PATH = deps/share/lua/$(LUA_VERSION)/?.lua;deps/share/lua/$(LUA_VERSION)/?/init.lua;lua/?.lua;lua/?/init.lua;$(shell lua$(LUA_VERSION) -e 'print(package.path)')
```

Reply:
Works. Thanks for the help.